dist/globals

```diff
@@ -0,0 +1,3 @@
+var tm4ApiUrl,
+    translationServerPort,
+    mergeServerPort;
```

Comment:
any reason not to name this one `globals.js`?

Reply:
this npm script removes all dist/*.js files and i was confused when i kept deleting `globals.js` unknowingly
https://github.com/ngageoint/hootenanny-ui/blob/hoot2x/package.json#L17

Reply:
i'm all for a better solution if we have one.

Reply:
maybe change the rm to `dist/*.min.js`?

Reply:
2️⃣ ⌚ 🔙 2️⃣ 🔙